src/sentry/static/sentry/app/stores/groupStore.tsx

```diff
@@ -2,58 +2,114 @@ import isArray from 'lodash/isArray';
 import isUndefined from 'lodash/isUndefined';
 import Reflux from 'reflux';
 
+import {Activity, Group} from 'app/types';
 import GroupActions from 'app/actions/groupActions';
 import IndicatorStore from 'app/stores/indicatorStore';
 import {t} from 'app/locale';
 
 function showAlert(msg, type) {
-  IndicatorStore.add(msg, type, {
+  IndicatorStore.addMessage(msg, type, {
     duration: 4000,
   });
 }
 
+// TODO(ts) Type this any better.
+type Change = [string, string, any];
```

Comment:
As of typescript 4 you can do named tuples 😄, not like it helps a ton but it's nice to know which of these values is supposed to be which on hover. The nice thing about this is you can replace rest params with a named tuple and the params will still be properly labeled.
```suggestion
type Change = [changeId: string, itemId:string, data:any];
```